### PR TITLE
[REVIEW] cugraph_dgl benchmarks

### PR DESCRIPTION
**Pull request by VibhuJawa**
This PR is a replacement of https://github.com/rapidsai/cugraph/pull/3078 . Needed a new PR to avoid a bunch of conflict resolution. 

Example command: 
```python
DASK_NUM_WORKERS=8 pytest bench_cugraph_dgl_uniform_neighbor_sample.py -k "MG and fanout_10_25 and rmat_26_8" gi--benchmark-save='8_rmat_26_8.json'
```

### CODE REVIEW

**Comment by VibhuJawa** [comment]
rerun tests

File: benchmarks/cugraph-dgl/pytest-based/bench_cugraph_dgl_uniform_neighbor_sample.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+import time
+import os
+import pytest
+import numpy as np
+import cupy as cp
+# Facing issues with rapids-pytest-benchmark plugin
+# pytest-benchmark.
+import pytest_benchmark
+benchmark = pytest_benchmark.plugin.benchmark
```
Comment (rlratzel) [inline benchmarks/cugraph-dgl/pytest-based/bench_cugraph_dgl_uniform_neighbor_sample.py:27]: This shouldn't be necessary. `pytest` will always make the `benchmark` fixture available if the plugin is installed.

**Review by rlratzel — APPROVED** [review]
LGTM, but I did have one minor suggestion.

File: benchmarks/cugraph-dgl/pytest-based/bench_cugraph_dgl_uniform_neighbor_sample.py (hunk 1 above)
Comment (VibhuJawa) [reply benchmarks/cugraph-dgl/pytest-based/bench_cugraph_dgl_uniform_neighbor_sample.py]: Resolved by https://github.com/rapidsai/cugraph/pull/3092/commits/459a190d25966daa9239b9c6a4eb510ceec34aa5

**Comment by VibhuJawa** [comment]
@rlratzel , All changes have been addressed , please feel free to merge.

**Comment by rlratzel** [comment]
@gpucibot merge

**Comment by VibhuJawa** [comment]
rerun tests. 

Failure Reason: `FAILED: 1 or more tests in cugraph-service`

**Comment by VibhuJawa** [comment]
rerun tests

Reason: FAILED: 1 or more tests in cugraph-service

**Comment by VibhuJawa** [comment]
rerun tests

**Comment by BradReesWork** [comment]
@gpucibot merge

**Review by ajschmidt8 — APPROVED** [review]

**Comment by ajschmidt8** [comment]
The `@gpucibot merge` commands on this PR are no longer valid now that we've switched to `/merge`.

But `/merge` can only be used by RAPIDS org members with `write` permissions or greater in a given repository.

**Pull request merged**